src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -1,13 +1,16 @@
 /* eslint-disable no-use-before-define */
 import {useNavigation, useRoute} from '@react-navigation/native';
 import PropTypes from 'prop-types';
-import {FlatList, Keyboard, StyleSheet, Text, View} from 'react-native';
+import * as React from 'react';
+import {ActivityIndicator, FlatList, Keyboard, StyleSheet, Text, View} from 'react-native';
+
 
 import React, {useState} from 'react';
 import Accordion from 'react-native-collapsible/Accordion';
 import LoadingWithoutModal from '../../../components/LoadingWithoutModal';
 import {Context} from '../../../context/Store';
-import {setFollow, setUnFollow} from '../../../service/profile';
+import useCreateChat from '../../../hooks/screen/useCreateChat';
+import {checkUserBlock, setFollow, setUnFollow} from '../../../service/profile';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a redundancy in the import of React. It's imported twice on lines 4 and 8. You should remove one of them.
```diff
- import * as React from 'react';
- import React, {useState} from 'react';
+ import React, { useState } from 'react';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->